pizzakit-plugin/includes/pizzakit.php

```diff
@@ -51,6 +55,14 @@ private static function handle_post()
 	}
 
 
+	public static function item_query_handler($data)
+	{
+		$json = file_get_contents(plugin_dir_path(__FILE__) . 'items_for_sale.json');
+		$decoded = json_decode($json, true);
+		$items = $decoded['menu'];
+		wp_send_json($items);
```

Comment:
Something like this maybe. Check that it actually works first.

```suggestion
		global $wpdb;
		$sql = "SELECT * FROM wp_items";
		$items = $wpdb->get_results($sql);
		wp_send_json($items);
```

Reply:
Fungerade typ, men wordpress skickade alla värden som strings. Roligt nog är JS typing system lika idiotiskt som php så allt fungerade när jag gjorde om några `===` till `==`.